<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestRunConfigurationWidgetClass</class>
 <widget class="QWidget" name="TestRunConfigurationWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test run configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="m_path_to_test_executable_line_edit"/>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="m_analyze_code_coverage_check_box">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Analyze code coverage</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QWidget" name="m_code_coverage_configuration_widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Path to configuration file</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Path to test results</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="m_browse_test_results_btn">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="m_path_to_test_results_line_edit"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="m_browse_test_executable_btn">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Path to test executable</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QPushButton" name="m_run_test_btn">
     <property name="text">
      <string>Run test</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>m_path_to_test_executable_line_edit</tabstop>
  <tabstop>m_browse_test_executable_btn</tabstop>
  <tabstop>m_path_to_test_results_line_edit</tabstop>
  <tabstop>m_browse_test_results_btn</tabstop>
  <tabstop>m_analyze_code_coverage_check_box</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>m_run_test_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
